uwh-refbox/src/main.rs

```diff
@@ -140,53 +140,259 @@ fn main() -> std::result::Result<(), Box<dyn std::error::Error>> {
+                            .stroke(Some(game_color))
+                            .translate(Point::new(80, 2)),
+                    ),
+                    GamePeriod::OvertimeHalfTime => display.draw(
+                        Font8x15::render_str("O/T HALF TIME")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(76, 2)),
+                    ),
+                    GamePeriod::OvertimeSecondHalf => display.draw(
+                        Font8x15::render_str("O/T 2ND HALF")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(80, 2)),
+                    ),
+                    GamePeriod::PreSuddenDeath => display.draw(
+                        Font8x15::render_str("PRE-SUDDEN DEATH")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(64, 2)),
+                    ),
+                    GamePeriod::SuddenDeath => display.draw(
+                        Font8x15::render_str("SUDDEN DEATH")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(80, 2)),
+                    ),
+                    _ => display.draw(
+                        Font8x15::render_str("ERROR_1")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(64, 2)),
+                    ),
+                }
+            }
+            _ => {
+                display.draw(
+                    Font16x31::render_str(&secs_to_time_string(state.secs_in_period))
+                        .stroke(Some(game_color))
+                        .translate(Point::new(108, 33)),
+                );
+                match state.current_period {
+                    GamePeriod::FirstHalf => {
+                        display.draw(
+                            Font8x15::render_str("1ST")
+                                .stroke(Some(game_color))
+                                .translate(Point::new(72, 33)),
+                        );
+                        display.draw(
+                            Font8x15::render_str("HALF")
+                                .stroke(Some(game_color))
+                                .translate(Point::new(68, 48)),
+                        );
+                    }
+                    GamePeriod::SecondHalf => {
+                        display.draw(
+                            Font8x15::render_str("2ND")
+                                .stroke(Some(game_color))
+                                .translate(Point::new(72, 33)),
+                        );
+                        display.draw(
+                            Font8x15::render_str("HALF")
+                                .stroke(Some(game_color))
+                                .translate(Point::new(68, 48)),
+                        );
+                    }
+                    _ => display.draw(
```

Comment:
What about other game periods? We can have ref timeouts during any period...

Reply:
This is another case where I think we don't want a default

Reply:
I'm not sure I understand what you mean. These lines correspond to a section that is just displaying the text for 1st half or 2nd half in the case of a timeout. Do you mean `_ =>` when you say default?

Reply:
yes, the `_` arm of the match is called the default

Reply:
Can you address the first part of my response please?